mindmap/static/html/mindmap.html

```diff
@@ -1,24 +1,66 @@
 {% load i18n %}
-{% if error_message %}
-  <p>{% trans "The mind map cannot be rendered due to the following error" %}: <b>{{error_message}}</b></p>
-  <p>{% trans "Please contact the instructor or an administrator" %}.</p>
-{% else %}
-  <link type="text/css" rel="stylesheet" href="https://unpkg.com/jsmind@latest/style/jsmind.css" />
-  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.11.5/css/jquery.dataTables.css" />
-
-  <div>
-    <div id="jsmind_container_{{xblock_id}}" class="jsmind_container"></div>
+
+{% if editable and can_submit_assignment or not in_student_view %}
```

Comment:
where did the error message go?

Reply:
I think the error message was added way back when using django storages, now it's not needed anymore. But not sure, as I've not been involved so often in this development

Reply:
Yes, the `error_message` variable is no longer in the context due to the change mentioned by @Ian2012.